app/javascript/src/components/login/signup/When2Meet.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
```

Comment:
Nice, I really like how the key looks on this component, but why did you decide to make an entirely new component rather than just build off of the `Availability` component? There is a very good practice in programming called "DRY" (Don't Repeat Yourself), so, having two files doing the same thing goes against that practice and is very cluttering. So move your changes to `availability/index.jsx` once you can. 

Moreover, having a component named `When2Meet` goes again React practices as it is a number on a component name.